test/unit/specs/components/AudioTrack/v-box-layout.spec.js

```diff
@@ -0,0 +1,50 @@
+import { render } from '@testing-library/vue'
+import Vuei18n from 'vue-i18n'
+import { createLocalVue } from '@vue/test-utils'
+
+import { getAudioObj } from '~~/test/unit/fixtures/audio'
+
+import VBoxLayout from '~/components/VAudioTrack/layouts/VBoxLayout.vue'
+
+const enMessages = require('~/locales/en.json')
+
+const i18n = new Vuei18n({
+  locale: 'en',
+  fallbackLocale: 'en',
+  messages: { en: enMessages },
+})
+
+describe('VBoxLayout', () => {
+  let options = null
+  let localVue
+  let props = {
+    audio: getAudioObj(),
+    size: 'm',
+  }
+
+  beforeEach(() => {
+    localVue = createLocalVue()
+    localVue.use(Vuei18n)
+    options = {
+      propsData: props,
+      mocks: { $nuxt: { context: { i18n } } },
+      localVue,
+      i18n,
+    }
+  })
+
+  it('renders audio title, license and category in v-box-layout', () => {
+    props.audio.category = 'music'
+    const screen = render(VBoxLayout, options)
+    screen.getByText(props.audio.title)
+    screen.getByLabelText('Attribution-NonCommercial-Share-Alike')
+    screen.getByText('Music')
```

Comment:
These are not `expect()` calls. Does that still count as a test?

Reply:
@dhruvkb , yes! `getBy` queries in testing library throw an error if they can't find the node.

> getBy...: Returns the matching node for a query, and throw a descriptive error if no elements match or if more than one match is found (use getAllBy instead if more than one element is expected).

From the testing library docs: https://testing-library.com/docs/queries/about